lib/billing_rate.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "yaml"
+
+class BillingRate
+  def self.rates
+    @@rates ||= YAML.load_file("config/billing_rates.yml")
+  end
+
+  def self.from_resource_properties(resource_type, resource_family, location)
+    rates.find {
+      _1["resource_type"] == resource_type && _1["resource_family"] == resource_family && _1["location"] == location
```

Comment:
Not required for now, we can think about when have too much rates. Hash map might decrease look up to O(1) instead of O(N)